<?xml version="1.0" ?>
<launch>
  <arg name="robot_ip" default="172.16.0.2"/>
  <arg name="gripper_homed" default="false" />

  <include file="$(find franka_gripper)/launch/franka_gripper.launch" >
    <arg name="robot_ip" value="$(arg robot_ip)" />
    <arg name="stop_at_shutdown" value="true" />
  </include>

  <node name="my_gripper_node" pkg="franka_example_controllers" type="my_gripper_node" respawn="false" output="screen" >
    <rosparam param="gripper_homed" subst_value="true">$(arg gripper_homed)</rosparam>
    <!-- <remap from="~leader/joint_states" to="$(arg leader_id)/franka_gripper/joint_states" />
    <remap from="leader/homing" to="$(arg leader_id)/franka_gripper/homing" /> -->
    <remap from="follower/homing" to="/franka_gripper/homing" />
    <remap from="follower/grasp" to="/franka_gripper/grasp" />
    <remap from="follower/move" to="/franka_gripper/move" />
    <remap from="follower/stop" to="/franka_gripper/stop" />
  </node>
</launch>